packages/react/test/specs/behaviors/treeItemAsListItemBehavior-test.tsx

```diff
@@ -0,0 +1,10 @@
+import { treeItemAsListItemBehavior } from 'src/lib/accessibility'
+
+describe('TreeItemAsListItemBehavior', () => {
+  describe('role', () => {
+    test(`is 'none' if not a leaf`, () => {
+      const expectedResult = treeItemAsListItemBehavior({ hasSubtree: true })
```

Comment:
Misleading variable name. This comment applies to other tests as well.
```suggestion
      const behavior = treeItemAsListItemBehavior({ hasSubtree: true })
```

Reply:
all the custom behavior tests have the same variable name. I'd leave it like this for now.

Reply:
for consistency I made this one boolean as well. what do you think?